.env.example

```
@@ -0,0 +1,31 @@
+# 👋 Welcome, we're glad you're setting up an installation of Talawa-admin. Copy this 
+# file to .env or set the variables in your local environment manually.
+
+
```

dotenv-linter (3.3.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)